release-workflow.md

```diff
@@ -14,7 +14,7 @@ To learn more about that, please read the details in [the GitHub documentation](
   - Step 1: Please install cffconvert using the following command: `python3 -m pip install --user cffconvert` (details here: https://pypi.org/project/cffconvert/)
   - Step 2: Run the following command: `cffconvert --validate`
   - Step 3: Once validated, create `.zenodo.json` file using the following command: `cffconvert --format zenodo > .zenodo.json`
-
+  - Step 4: Create the pull request with the change -> review and merge it in the repository
```

Comment:
```suggestion
  - Step 4: Create the pull request with the change -> review and merge it in the repository
  
```

Adding a newline here to stop the following paragraph being part of this bullet point.